code/modules/paperwork/paper_bundle.dm

```diff
@@ -77,29 +77,23 @@
 	add_fingerprint(usr)
 	return
 
-/obj/item/paper_bundle/proc/burnpaper(obj/item/lighter/P, mob/user)
+/obj/item/paper_bundle/proc/burnpaper(obj/item/heating_object, mob/user)
 	var/class = "<span class='warning'>"
 
-	if(P.lit && !user.restrained())
-		if(istype(P, /obj/item/lighter/zippo))
-			class = "<span class='rose'>"
+	if(istype(heating_object, /obj/item/lighter/zippo))
+		class = "<span class='rose'>"
 
-		user.visible_message("[class][user] holds [P] up to [src], it looks like [user.p_theyre()] trying to burn it!</span>", \
-		"[class]You hold [P] up to [src], burning it slowly.</span>")
+	user.visible_message("[class][user] holds [heating_object] up to [src], it looks like [user.p_theyre()] trying to burn it!</span>", "[class]You hold [heating_object] up to [src], burning it slowly.</span>")
```

Comment:
Can you move the spans themselves into the visible messages, and just pull out the actual classes themselves (e.g. warning, rose)

Reply:
reviews addressed